events/wmt20.md

```diff
@@ -225,7 +225,7 @@ Full results of the shared tasks: [*Findings of the 2020 Conference on Machine T
 ### News translation
 
 #### → English
-The human evaluation was based on segment ratings with document context.
+The human evaluation used reference-based (monolingual) segment ratings with document context.
 
 | Language pair | System | Average score | Average z-score (standard score) |
```

Comment:
Could you add an explanation regarding what Average score and Average z-score are, and how they differ from each other? Maybe these explanations should appear in a different page? What do you think?

Also, have these systems won by their Average score or by their Average z-score? Perhaps it's best to clarify. Have you added only the average scores of the winning systems?

Do you think a bold format for the winning systems would make the table clearer?

Reply:
As I understand it, the only systems listed are the winning systems (for each language pair).

So we don't need to bold, because they'd all be bold.

But we do need to say that, so it's clear.

Reply:
As Adam already stated, the results list only the winning systems. Moreover, these are only official results of the conference. Each year, there may be other types of results as well, e.g., WMT 2021 findings also include source-based and contrastive source-based results of the conference. 
But I haven't included those not to make the overview too complicated for the viewers because those results may need more complex explanations equal to those presented in the findings. 

To make it clear, I suggest writing a short description that can follow the `News translation` title. 

Something like this

> ### News translation
> Official results of WMT20 News Translation Task. Winner systems are listed according to the average z-score and average score. 

Or have a separate description for each translation type

> #### → English
> Official results of WMT20 News Translation Task for translation into English. Winner systems are listed according to the average z-score and average score. The human evaluation used monolingual segment ratings with document context. In this configuration, the human assessors should rate the system output by comparing it to the human-generated reference without considering the source input.


As for adding an explanation regarding what Average score and Average z-score are, I suggest having a separate page for several terms used in WMT, as listing those explanations on each year's conference page will make the pages look heavy with text. For example, the explanation regarding average score and z-score is something like this

> Winner systems are listed according to the average z-score and average score. The average z-score shows how many standard deviations below or above the mean a raw score is. To compute it, human assessment scores for translations are first standardized according to each human assessor’s overall mean and standard deviation score. Average z-scores for individual segments belonging to a given system are then computed before the final overall direct assessment score for that system is computed as the average standard of its segment scores. Results are also reported for average scores for systems, computed in the same way but without any score standardization applied.

And I think there are other terms as well that may need some explanation, such as direct assessment, relative ranking (this was used during previous WMTs), segment ratings with and without document context or in a full document, etc.

Let me know what you think.

Reply:
> To make it clear, I suggest writing a short description that can follow the News translation title.
> Something like this
> ### News translation
> Official results of WMT20 News Translation Task. Winner systems are listed according to the average z-score and average score.

I would omit "Official results of WMT20 News Translation Task" as that is clear.
Naming them "Winner systems" is helpful to understand what follows too.



> As for adding an explanation regarding what Average score and Average z-score are, I suggest having a separate page for several terms used in WMT

Totally, we shouldn't add repeated info to all WMT pages.

Would you explain these terms in our [WMT](wmt.md) page or create a new terminology page related to WMT shared tasks?

Reply:
I guess we can add the terms to the WMT page, as I think there are only several terms to explain, and there's no need to create a separate page. Let me know what you think, and I'll create a separate PR for the terms.

Reply:
Sure, go ahead!

Please make sure to link the articles in their corresponding key terms so that readers can go back and forth between them.